openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -96,6 +96,10 @@ OpenAssessment.BaseView.prototype = {
                     $toggleButton.attr('aria-expanded', 'true');
                     $container.addClass('is--showing');
                 }
+
+                if ($container.hasClass('is--initially--collapsed ')) {
+                    $container.removeClass('is--initially--collapsed ');
```

Comment:
I don't think there is any harm in just always calling removeClass, regardless of whether or not is--initially--collapsed is present. it will be a no-op if the class is not there.